zebra-scan/src/service/tests.rs

```diff
@@ -0,0 +1,79 @@
+//! Tests for ScanService.
```

Comment:
This tests are great, thank you creating them. One thing i noticed is that we can run this tests without the `shielded-scan` feature flag. Is this something we might want to change?

Reply:
I think it's okay for the zebra-scan tests to always run and for the `shielded-scan` feature to enable integration tests with `zebrad`.